nncf/onnx/graph/node_utils.py

```diff
@@ -48,12 +44,11 @@ def get_bias_value(node_with_bias: NNCFNode, model: onnx.ModelProto) -> np.ndarr
     :return: The bias value that is applied to the output tensor of the node's operation.
     """
     onnx_graph = ONNXGraph(model)
-    onnx_node = onnx_graph.get_node_by_name(node_with_bias.node_name)
-    bias_port_id = onnx_graph.get_bias_tensor_port_id(onnx_node)
-    bias_input_name = onnx_node.input[bias_port_id]
-    if onnx_graph.has_tensor(bias_input_name):
-        return onnx_graph.get_tensor_value(bias_input_name)
-    node = onnx_graph.get_nodes_by_output(bias_input_name)[0]
+    assert node_with_bias.layer_attributes.has_bias()
+    bias_name = node_with_bias.layer_attributes.bias_attrs["name"]
+    if onnx_graph.has_tensor(bias_name):
```

Comment:
What if it hasn't this tensor? Is it saying about incorrect `NNCFGraph`?

Reply:
This function relies that `NNCFGraph` is correct

Reply:
True, I will remove this check

Reply:
Done